docs/Request.md

```diff
@@ -2,23 +2,62 @@
 
 ## Request
 The first parameter of the handler function is `Request`.<br>
-Request is a core Fastify object containing the following fields:
+Request is a core Fastify object containing the following properties:
 - `query` - the parsed querystring
-- `body` - the body
+- `body` - the body content, properly parsed by the Content-Type value
 - `params` - the params matching the URL
-- `headers` - the headers
+- `headers` - the parsed headers
 - `raw` - the incoming HTTP request from Node core *(you can use the alias `req`)*
 - `id` - the request id
 - `log` - the logger instance of the incoming request
 
+**Example**
+
 ```js
-fastify.post('/:params', options, function (request, reply) {
-  console.log(request.body)
-  console.log(request.query)
-  console.log(request.params)
-  console.log(request.headers)
-  console.log(request.raw)
-  console.log(request.id)
+fastify.post('/resource/:section/:id', options, function (request, reply) {
+  console.log('parsed body params\n' + JSON.stringify(request.body, null, 2))
+  console.log('parsed query params\n' + JSON.stringify(request.query, null, 2))
+  console.log('routing params\n' + JSON.stringify(request.params, null, 2))
+  console.log('parsed headers\n' + JSON.stringify(request.headers, null, 2))
+  console.log('request inner id' + request.id)
+  console.log('raw request instance\n' + request.raw)
```

Comment:
This prints `[Object object]`, I think. If we want to show the output, is this output really needed?